### PR TITLE
link to uganda district cases worked on

### PR DESCRIPTION
**Pull request by lynnnakiganda**

### CODE REVIEW

File: data/districts_data.js [hunk 1]
```diff
@@ -105,7 +105,7 @@ var districts_data = {
...
...424707428 ], [ 31.675468885446126, -0.515333478889836 ], [ 31.667074529910558, -0.514544828319309 ], [ 31.658232726054514, -0.499655162605963 ], [ 31.659256646197818, -0.491451689518171 ], [ 31.665068414364381, -0.485202856970893 ], [ 31.666202734450401, -0.460673265037868 ], [ 31.662469486338317, -0.457705982019319 ], [ 31.657494469548116, -0.444255368052131 ], [ 31.6450912821076, -0.439739748171799 ], [ 31.621398699941857, -0.4441944432309 ], [ 31.596211131486037, -0.431310177710301 ], [ 31.590528906212292, -0.443021869703721 ], [ 31.577766198958997, -0.453140260303431 ], [ 31.571514000522264, -0.463115119992555 ], [ 31.569550110633354, -0.476898540757126 ], [ 31.563916984443175, -0.482324059351246 ], [ 31.540162121762862, -0.495668507349817 ], [ 31.525529564789242, -0.497772595825829 ], [ 31.503734074013213, -0.516733431806072 ], [ 31.478175614869048, -0.543687892024146 ], [ 31.466744788209212, -0.547546812593445 ], [ 31.460677686548706, -0.554708148414823 ], [ 31.44526019555391, -0.554667936674596 ], [ 31.438517441597433, -0.562498213080521 ], [ 31.441223980421285, -0.57971865006342 ], [ 31.444336838857144, -0.586140738788156 ], [ 31.439953578850439, -0.5980865880018 ] ] ] } },
```
Comment (RonKbS) [inline data/districts_data.js:108]: Hi @lynnnakiganda ,

what changed precisely? On this line?

Comment (lynnnakiganda) [reply data/districts_data.js:108]: I changed kyotara to kyotera

Comment (lynnnakiganda) [reply data/districts_data.js:108]: I changed kyotara to kyotera

File: js/layers_fn.js [hunk 2]
```diff
@@ -1,151 +1,150 @@
-
```
Comment (RonKbS) [inline js/layers_fn.js:1]: @lynnnakiganda ,

Changes in this file aren't clear. Please see if you can paste back the origin file and then add any change that you're supposed to add. Do this in a commit and push to the same branch for this PR.

Comment (lynnnakiganda) [reply js/layers_fn.js:1]: okay

Comment (lynnnakiganda) [reply js/layers_fn.js:1]: i can't change the same thing in this branch because its seeing like nothing changed

**Pull request closed without merging**